universal-application-tool-0.0.1/app/views/admin/questions/QuestionEditView.java

```diff
@@ -284,10 +298,12 @@ private DomContent formQuestionTypeSelect(QuestionType selectedType) {
    * question definition's repeater id.
```

Comment:
Question (blocking): Should this field be disabled? This is the case where someone is editing a question and can't actually change the repeater, right? If this requires a bigger change than makes sense for this PR, just add a TODO.

Reply:
I'll do this in a follow-up PR.